content/english/jsappybird/add-jump-sound.md

```diff
@@ -10,6 +10,8 @@ Let's add sounds to the bird while it is jumping
 Add the below command in the file [File: create-game.js]
     jumpSound.play();
 
+{{% notice tip %}}
```

Comment:
All these {{% notice tip %}} will need a corresponding {{%/ notice %}} tag, similar to HTML tags. Please see other workshops for examples